python/ray/_private/ray_logging/filters.py

```diff
@@ -1,10 +1,13 @@
 import logging
 import ray
+import time
 from ray._private.ray_logging.constants import LogKey
 
 
 class CoreContextFilter(logging.Filter):
     def filter(self, record):
+        setattr(record, LogKey.TIMESTAMP_NS.value, time.time_ns())
```

Comment:
maybe we should derive the value of `timestamp_ns` from the record instead of calling `time.time_ns()` to avoid inconsistency. Example: https://github.com/ray-project/ray/pull/46074

Reply:
Good call, thanks for the suggestion!